sys-block/sas3ircu/sas3ircu-16.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit mount-boot
+
+DESCRIPTION="LSI MPT-SAS3 controller management tool"
+HOMEPAGE="https://www.broadcom.com/products/storage/host-bus-adapters/sas-9300-8e#downloads"
+LICENSE="LSI"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~ppc64 ~x86 ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE="efi doc"
+RESTRICT="strip fetch mirror"
+DEPEND=""
+RDEPEND=""
```

Comment:
Empty `DEPEND/RDEPEND` variables are pointless and should be left out.

This probably should have `BDEPEND="app-arch/unzip` since it needs to unpack a .zip

Reply:
Done.